### ISSUE TITLE
Document and add examples for vertical alignment

### ISSUE THREAD

**Issue by Darkyenus** [sile-typesetter/sile#577]
As discussed on Gitter, aligning content on the page vertically is not trivial (at least without any in-depth knowledge of internals), but it should be possible to do.

SILE book, as the main documentation, should mention how to do this. Several cases that should be handled and described come to mind:
1. Empty page, align everything down, possibly with some fixed offset from the bottom
2. Empty page, center everything (posibly with some offset)
3. *1* and *2*, but with some content already on the page. This may not always be possible to satisfy perfectly (centering a paragraph when existing content would intersect it), but there should be a way to get at least close

Good explanations should explain why some approach works, not just be a code snippet, so it is easier to extend/modify/fix when it breaks.

If any of the above is not (currently?) possible, it should be mentioned as well, with the explanation why.

And, of course, all of this does not have to be added at once. Scarce documentation is better than no documentation.

**Comment by Omikhleia** [comment]
I am not sure what the discussions on the (old?) Gitter was, @Darkyenus -- and what were the problems at stakes.

This being said, I proposed some enhancements in #1813, feel free to comment there if need be. In particular, I assumed there were some approaches that failed, since you said:
> Good explanations should explain why some approach works, not just be a code snippet, so it is easier to extend/modify/fix when it breaks.

But I am not sure what they could have been, and if the proposed changes to the manual are sufficient.

For me they are, and the various "cases" described above can just be derived from the proposed example.

 1. Align everything down, possibly with some fixed offset from the bottom:
     ```
     \hbox{}% This is an empty initial line
     \vfill
     Aligned down some fixed offset from the bottom
     \skip[height=2cm]
     \break
     ```

  2. Center everything
 
     ```
     \hbox{}% This is an empty initial line
     \vfill
     Centered vertically
     \vfill
     \break
     ```

  3. 1 and 2... 
      > but with some content already on the page. This may not always be possible to satisfy perfectly (centering a paragraph when existing content would intersect it), but there should be a way to get at least close

     Er. I am at loss understanding what is requested here, and what the use case means. :confused:

**Comment by Darkyenus** [comment]
Sorry, it has been too long and I don't remember anything about this and I haven't used Sile in a long time either, so I won't be much help. Feel free to close this issue. It is possible that this is no longer relevant, if the Sile internals changed a lot in the last 4 years.

**Comment by Omikhleia** [comment]
Thanks for the feedback. The issue will be closed when #1813 gets merged then -- And perhaps case (3) is not fully addressed, but if no one remembers what it was about exactly, well it's probably still better than nothing.
